lib/pacemaker/libpacemaker_private.h

```diff
@@ -25,8 +25,21 @@ struct resource_alloc_functions_s {
     gboolean(*create_probe) (pe_resource_t *, pe_node_t *, pe_action_t *, gboolean, pe_working_set_t *);
     void (*internal_constraints) (pe_resource_t *, pe_working_set_t *);
 
+    /*!
+     * \internal
+     * \brief Apply a colocation's score to node weights or resource priority
+     *
+     * Given a colocation constraint, apply its score to the dependent's
```

Comment:
Does this function only apply the score to the dependent's allowed node weights, or could it affect the primary if influence=true?

Reply:
The dependent's scores can later affect the primary, but this method only touches the dependent.